linux/devdoc/async_socket_linux_requirements.md

```diff
@@ -185,12 +149,8 @@ MOCKABLE_FUNCTION(, void, async_socket_close, ASYNC_SOCKET_HANDLE, async_socket)
 
 **SRS_ASYNC_SOCKET_LINUX_11_037: [** `async_socket_close` shall wait for all executing `async_socket_send_async` and `async_socket_receive_async` APIs. **]**
```

Comment:
>all executing `async_socket_send_async` and `async_socket_receive_async` [](http://example.com/codeflow?start=73&length=72)

This "wait for all" how does it work? it looks like some sort of "counting" would be necessary. 

How is "all" determined?

Reply:
We have a pending call count that letsthis function know that all the executing send_async/receive_async calls.